site/docs/components/static-list/examples.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+
+  <LivePreview componentName="static-list" exampleName="Multiple" >
+
+## Multiple
+
+`Static list` group showing multiple `StaticListItems` stacked together.
+
+  </LivePreview>
+
+  <LivePreview componentName="static-list" exampleName="AdditionalLabel" >
+
+## Additional label
+
+You can add additional labels inside `StaticListItem` to provide extra context using the [`Label`](../label) and [`Stack Layout`](../stack-layout) components.
+
+  </LivePreview>
+
+  <LivePreview componentName="static-list" exampleName="WithIcons" >
+
+## With Icons
+
+`StaticListItem` with descriptive icon grouped together to organize related content. You can change the default descriptive icon with other icons based on your use-case.
+
+  </LivePreview>
+
+  <LivePreview componentName="static-list" exampleName="WithButtons" >
+
+## With Buttons
+
+You can use [`Button`](/salt/components/button) to allow custom interactivity within the `StaticListItem`.
```

Comment:
Can we make this less generic? (custom interactivity). Maybe this should be more about adding actions to the list item with an example relevant to the example with it.

Reply:
We had kept it generic because the examples are also generic
Do you have a suggestion on the description @ivan-calderon ?

Reply:
@ThusharaJ07, @joshwooding I just added [a new example in Figma](https://www.figma.com/design/pXXcdSsLVdsSuFgpnfj9L5/Static-list?node-id=9658-35787&t=0p16WLhg8GHo540w-1) (in orange). Please LMK if that makes more sense. If so, also let me know if we need to revisit the other examples. TY!